doc/util/chpl2rst.py

```diff
@@ -215,8 +217,18 @@ def gen_rst(handle):
                     rstline = rstline[indentation:]
                 else:
                     rstline = rstline.lstrip(' ')
-
-            output.append(rstline)
+            # Special case for multi line comment in learnChapelInYMinutes
+            if(islearnChapelInYMinutes and ('multi-line comment' in rstline)):
+                output.append(' '*indentation + '/*')
+                output.append(rstline)
+                output.append(' '*indentation + '*/')
+                if(foundCommentExample):
+                    print("Error: Found more than one line containing \"multi-line comment\" in \
```

Comment:
Should this be a thrown exception instead of a print and then exit?

Reply:
I don't know Python well enough to be able to say the difference, but I found the error sufficiently clear to understand when introducing the error condition and testing it (as an end-user).  So, for me, this is good enough and probably isn't worth  more effort, but I probably don't understand the tradeoffs.

Reply:
I followed the pattern from another case in the script where if the output is not valid then the script prints a message and exits. Since exiting is the behavior we want we really won't be catching the exception anywhere so even though it would look better stylistically the behavior would be the same.